### DISCUSSION TITLE
Better to move feature request from issue to discussion

### DISCUSSION THREAD

**Discussion by flashlab** [VueTorrent/VueTorrent#2064] [General]
Vuetorrent is really cool and alomost the best choice besides the official one, for now it can definitely satisfiing most users' demand in regular task. 
Considering increasing feature request, it'll be better to encourage users to put forward any none core feature request here, before it become bloat. Cause others can vote the priority here. And contributers may put more energy on security and bugfix.

**Comment by Larsluph** [comment]
I like the suggestion but I haven't really used discussions a lot on GitHub TBH.

The idea I have of Discussions is more of a blog-like experience, where you can create topics and comment on it.

Tracking tasks seems harder and I think discussions are not made for that particular use case. It might be worth it to do some research on that.
